Src/BootCamp.Chapter/Application.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Application
+    {
+        readonly CredentialsManager manager;
+        readonly UI ui;
+
+        public Application(string credentialsFile)
+        {
+            manager = new CredentialsManager(credentialsFile);
```

Comment:
Please rename to CredentialsManager. Because right now- certainly you have only one component which does something that could be called managing, but that could change quite easily. Also if you come back to the code after a month or two, you will have a hard time remembering what this manager really is for.